README.md

```diff
@@ -17,6 +17,10 @@ should be able to preview your changes at http://localhost:4567
 
     bundle exec middleman server
 
+Dockerfile is included for processing PHP content. To run the PHP server execute the following from the root repo directory:
```

Comment:
Wasn't aware of the option! I've refactored to use middleman-php...thanks for this!